src/Nethermind/Nethermind.Config/ExitCodes.cs

```diff
@@ -18,6 +18,7 @@ public static class ExitCodes
     public const int DuplicatedOption = 105;
     public const int UnrecognizedOption = 106;
     public const int ForbiddenOptionValue = 107;
+    public const int MissingChainspecEipConfiguration = 107;
```

Comment:
Not sure if it's intentional, but just in case

```suggestion
    public const int MissingChainspecEipConfiguration = 108;
```